code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4976,3 +5014,22 @@
 	reagents.add_reagent(MEDCORES, 6)
 	reagents.add_reagent(SODIUMCHLORIDE, 6)
 	reagents.add_reagent(NUTRIMENT, 4)
+
+
+/obj/item/weapon/reagent_containers/food/snacks/lasagna
+	name = "lasagna"
+	desc = "A carefully stacked trayful of meat, tomato, cheese, and pasta. Favorite of cats."
+	icon_state = "lasagna"
+	bitesize = 3
+	storage_slots = 1
+	food_flags = FOOD_ANIMAL //cheese
+
+/obj/item/weapon/reagent_containers/food/snacks/lasagna/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 12)
+	reagents.add_reagent(TOMATOJUICE, 15)
+
+/obj/item/weapon/reagent_containers/food/snacks/lasagna/can_hold(obj/item/weapon/W) //GREAT SCOTT!
+	if(is_type_in_list(W, list(/obj/item/device/transfer_valve, /obj/item/toy/bomb, /obj/item/weapon/plastique, /obj/item/device/fuse_bomb, /obj/item/weapon/grenade, /obj/item/device/onetankbomb)))
+		return TRUE
```

Comment:
> one tank bomb

do you have any idea how fucking big those are?!

Reply:
I'd say about half as big as a two tank bomb

Reply:
Oh and this should be a static list

Reply:
Also @N3X15 the lasagna and one-tank bombs are exactly the same size, so I don't think there's gonna be a problem with that. There's not much advantage to it besides stale 90's kid references.